src/OrchardCore.Modules/OrchardCore.Localization/Views/AdminCulturePickerContainer.cshtml

```diff
@@ -0,0 +1,33 @@
+@using OrchardCore.Localization
+@using OrchardCore.Environment.Shell
+@inject ShellSettings ShellSettings
+
+@{
+    var selectedCultureName = Model.CurrentCulture.Name;
+}
+<select id="oc-admin-culture-picker" class="form-select me-3"
+    data-cookie-name="@(LocalizationCookieName.AdminCulturePrefix + ShellSettings.VersionId)"
+    data-cookie-path="@(Context.Request.PathBase.HasValue ? Context.Request.PathBase : "/")"
```

Comment:
We could check the `ShellSettings.Name` if it's a default shell or not

Reply:
The default shell may have a non empty url prefix.

A non default tenant may have an empty url prefix.

And PathBase reflects the url prefix in a well escaped form.

We only need to check if it has a value.